src/View/ViewableData.php

```diff
@@ -619,7 +619,7 @@ public function Me()
      * project directory.
      *
      * @return string URL to the current theme
-     * @deprecated 4.0.0:5.0.0 Use $resourcePath or $resourceURL template helpers instead
+     * @deprecated 4.0.0 Use $resourcePath or $resourceURL template helpers instead
```

Comment:
```suggestion
     * @deprecated 4.0.0 Use ModuleResourceLoader::resourcePath() or ModuleResourceLoader::resourceURL() instead
```
This is more correct as it indicates where the methods are which need to be called.
Also update the notice message.

Reply:
Updated